service/src/main/java/com/ge/predix/acs/attribute/connectors/ResourceAttributeReader.java

```diff
@@ -0,0 +1,5 @@
+package com.ge.predix.acs.attribute.connectors;
+
+public interface ResourceAttributeReader extends AttributeReader {
```

Comment:
Do we anticipate adding more functionality to this interface in the immediate future?

If so, maybe put a comment saying that it shouldn't be refactored out?

If not, should we delete this interface and replace `implements ResourceAttributeReader` with `implements AttributeReader` everywhere?